Integrations/EWSv2/EWSv2.py

```diff
@@ -1150,25 +1157,32 @@ def parse_incident_from_item(item, is_fetch):
 
 
 def fetch_emails_as_incidents(account_email, folder_name):
-    start_time = EWSDateTime.now(tz=EWSTimeZone.timezone('UTC'))
     last_run = get_last_run()
 
     try:
         account = get_account(account_email)
         last_emails = fetch_last_emails(account, folder_name, last_run.get(LAST_RUN_TIME), last_run.get(LAST_RUN_IDS))
 
-        ids = []
+        ids = deque(last_run.get(LAST_RUN_IDS, []), maxlen=LAST_RUN_IDS_QUEUE_SIZE)
```

Comment:
Same as previous comments, here it is pretty much guaranteed that `None` will be returned after the content update. Use the same method as above to ensure that `None` is replaced with `[]`.

Reply:
fixed in https://github.com/demisto/content/pull/4432/files#diff-f868b5eac3f32b0f92afbc8ec448c70fR886